### PR TITLE
Fix CulturePickerOptions settings

### PR DESCRIPTION
**Pull request by hishamco**
I think there's an issue with the `CulturePickerOptions` section name, I did a fix but I need to be 100% sure that is work fine, @jptissot when the `ContentCulturePickerController` is called, AFAIK there should be a culture picker in the site that trigger this, but I didn't see anything, did I miss something?

### CODE REVIEW

**Comment by hishamco** [comment]
Now I'm able to trigger the controller after adding `ContentCulturePicker` shape, time to test the PR ;)

**Comment by hishamco** [comment]
Everything looks good after testing

File: src/OrchardCore.Cms.Web/appsettings.json [hunk 1]
```diff
@@ -44,8 +44,10 @@
     //  "CreateContainer": true // Creates the container during app startup if it does not already exist.
     //},
     // Uncomment to configure content localization.
-    //"OrchardCore_ContentLocalization_CulturePicker": {
```
Comment (deanmarcussen) [inline src/OrchardCore.Cms.Web/appsettings.json:47]: This change is wrong.

The current section name is correct. It should match the `FeatureId`. Refer the `Manifest.cs` for more information.

We already spoke about this on your original pr.

Here you are confusing Features with namespaces, and class names. Which are not connected.

Comment (jptissot) [reply src/OrchardCore.Cms.Web/appsettings.json:47]: I think the current code is ok no ?

**Comment by hishamco** [comment]
> This change is wrong.

It's working fine Dean

> The current section name is correct. It should match the FeatureId. Refer the Manifest.cs to understand what a feature is.

I will have a look

> I think the current code is ok. No need for this PR

I will double check

File: src/OrchardCore.Cms.Web/appsettings.json (hunk 1 above)
Comment (hishamco) [reply src/OrchardCore.Cms.Web/appsettings.json:47]: > This change is wrong.

It's working fine Dean

> The current section name is correct. It should match the FeatureId. Refer the Manifest.cs to understand what a feature is.

I will have a look

> I think the current code is ok. No need for this PR

I will double check

**Comment by jptissot** [comment]
@deanmarcussen Can this be closed ? I am not familiar with the structure of the settings. If it's based on the Feature, then close, if it's based on module structure, then maybe consider this. From your comment, it sounds like the current implementation is correct and this can be closed.

**Comment by hishamco** [comment]
I'm not sure why we need to close this? I did similar thing in Markdown settings and the current PR works well

**Comment by deanmarcussen** [comment]
The current naming is correct and makes sense.

**Comment by hishamco** [comment]
Still wondering, what will happen if we add a new setting to `Content Localization`?!!

**Comment by jptissot** [comment]
From what I gather, it would use another section. Sections are configured per feature and not per namespace.

**Comment by hishamco** [comment]
IMHO settings should be grouped per module

**Pull request closed without merging**